jvm/src/main/scala/io/glutenproject/expression/ComplexTypeMergingExpressionTransformer.scala

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
Why this file is named as `ComplexTypeMergingExpressionTransformer`? Are these functions for complex type?

Reply:
These functions  have common trait ComplexTypeMergingExpression.